subtitue hardcoded cc for gcc to prevent 'cc1: error: unrecognized command line option "-fstack-protector-strong"'
author: Ruben van Dijk (University of Groningen)
--- Makefile.orig	2016-08-27 11:42:01.000000000 +0200
+++ Makefile	2016-08-27 11:36:53.000000000 +0200
@@ -34,7 +34,7 @@
 # These definitions are from config.sh (via /software/software/Perl/5.22.1-foss-2016a/lib/perl5/5.22.1/x86_64-linux-thread-multi/Config.pm).
 # They may have been overridden via Makefile.PL or on the command line.
 AR = ar
-CC = gcc
+CC = cc
 CCCDLFLAGS = -fPIC
 CCDLFLAGS = -Wl,-E
 DLEXT = so
